synapse/config/saml2.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ._base import Config
+
+
+class SAML2Config(Config):
+    def read_config(self, config):
+        self.saml2_config = config["saml2_config"]
+
+    def default_config(self, config_dir_path, server_name):
+        return """
+        saml2_config:
+            config_path: "%s/sp_conf.py"
+            idp_redirect_url: "http://%s/idp"
+        """ % (config_dir_path, server_name)
```

Comment:
I would prefer if saml2 was disabled by default, since not everyone is going to have saml. One way to do this would be to include the commented out config. It would also be nice to include some comments about what the config is for and links to documentation.